_data/applications.yml

```diff
@@ -45,6 +45,16 @@ ostree:
   description: |
     Upgrade OSTree-based systems in your browser.
 
+tukit:
```

Comment:
Can you please move this paragraph to the end? So that we keep the official/RH/third party grouping in the file?